public/assets/scss/_navbar.scss

```
@@ -8,7 +8,7 @@
     font-weight: 300;
 }
 %title-font-stack {
-    color: $gray-700;
+    color: $green;
     font-size: 1.2rem;
     font-weight: 400;
```

Can we update the font-weight to 500 (Semibold is used in the Figma file; though Semibold should be equivalent to 600, I think 500 works great when I tested it)
```suggestion
    font-weight: 500;
```